src/ResourceManager/Sql/Commands.Sql/ServerDnsAlias/Cmdlet/RemoveAzureSqlServerDnsAlias.cs

```diff
@@ -0,0 +1,158 @@
+		[Alias("Name")]
+		[ValidateNotNullOrEmpty]
+		public string DnsAliasName { get; set; }
+
+		/// <summary>
+		/// Gets or sets the name of the Azure Sql Server to use
+		/// </summary>
+		[Parameter(ParameterSetName = CmdletParametersParameterSet,
+			Mandatory = true,
+			HelpMessage = "The Azure Sql Server name.")]
+		[ValidateNotNullOrEmpty]
+		public override string ServerName { get; set; }
+
+		/// <summary>
+		/// Gets or sets the name of the resource group to use.
+		/// </summary>
+		[Parameter(ParameterSetName = CmdletParametersParameterSet,
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			HelpMessage = "The name of the resource group.")]
+		[ValidateNotNullOrEmpty]
+		public override string ResourceGroupName { get; set; }
+
+		/// <summary>
+		/// Server Dns Alias object to remove
+		/// </summary>
+		[Parameter(ParameterSetName = ObjectParameterSet,
+			Mandatory = true,
+			ValueFromPipeline = true,
+			HelpMessage = "The Server Dns Alias object to remove")]
+		[ValidateNotNullOrEmpty]
+		public Model.AzureSqlServerDnsAliasModel InputObject { get; set; }
+
+		/// <summary>
+		/// Gets or sets the resource id of the server dns alias
+		/// </summary>
+		[Parameter(ParameterSetName = ResourceIdParameterSet,
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			HelpMessage = "The resource id of Server Dns Alias object to remove")]
+		[ValidateNotNullOrEmpty]
+		public string ResourceId { get; set; }
+
+		/// <summary>
+		/// Defines whether it is ok to skip the requesting of rule removal confirmation
+		/// </summary>
+		[Parameter(HelpMessage = "Skip confirmation message for performing the action")]
+		public SwitchParameter Force { get; set; }
+
+		/// <summary>
+		/// Gets the entity to delete
+		/// </summary>
+		/// <returns>The entity going to be deleted</returns>
+		protected override IEnumerable<Model.AzureSqlServerDnsAliasModel> GetEntity()
+		{
+			return new List<Model.AzureSqlServerDnsAliasModel>()
+			{
+				ModelAdapter.GetServerDnsAlias(this.ResourceGroupName, this.ServerName, this.DnsAliasName)
+			};
+		}
+
+		/// <summary>
+		/// Apply user input.  Here nothing to apply
+		/// </summary>
+		/// <param name="model">The result of GetEntity</param>
+		/// <returns>The input model</returns>
+		protected override IEnumerable<Model.AzureSqlServerDnsAliasModel> ApplyUserInputToModel(IEnumerable<Model.AzureSqlServerDnsAliasModel> model)
+		{
+			return model;
+		}
+
+		/// <summary>
+		/// Deletes server dns alias
+		/// </summary>
+		protected override IEnumerable<Model.AzureSqlServerDnsAliasModel> PersistChanges(IEnumerable<Model.AzureSqlServerDnsAliasModel> entity)
+		{
+			ModelAdapter.RemoveServerDnsAlias(this.ResourceGroupName, this.ServerName, this.DnsAliasName);
+			return entity;
+		}
+
+		/// <summary>
+		/// Entry point for the cmdlet
+		/// </summary>
+		public override void ExecuteCmdlet()
+		{
+			if (!Force.IsPresent && !ShouldProcess(
```

Comment:
@maboja-msft at the design review, I think we talked about `Force` being used for the `ShouldContinue` prompt, but I don't see `ShouldContinue` in this cmdlet. Is it executed elsewhere? If not, you can see how it is used in this section on [`SupportsShouldProcess`](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#supportsshouldprocess).